content/info/awards/best-paper-awards.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Best Paper Awards
+layout: page
+permalink: /info/awards/best-paper-awards
+contact: [committee-name]@ieeevis.org
+---
+
+Best paper and honorable mention awards are selected per conference track by a dedicated committee of experienced community members. 
+Both awards recognize outstanding work from the pool of accepted papers in each conference track. Best paper committees use a variety of criteria to select the best paper including potential impact to the community, 
+the importance of any results obtained, technical challenges overcome, etc. 
+
+## VAST
+
+### Best Paper Award
+
+**VATLD: A Visual Analytics System to Assess, Understand and Improve Traffic Light Detection**
+<br/>
+Authors: Liang Gou, Lincan Zou, Nanxiang Li, Michael Hofmann, Shekar Arvind Kumar, Axel Wendt, and Liu Ren 
+
+### Best Paper Honorable Mention
+
+**In Search of Patient Zero: Visual Analytics of Pathogen Transmission Pathways in Hospitals**
+<br/>
+Authors: Tom Baumgartl, Markus Petzold_, Marcel Wunderlich, Markus Höhn, Daniel Archambault, Michael Lieser, Alexander Dalpke, Simone Scheithauer, Michael Marschollek, Vanessa M. Eichel, Nico T. Mutters, and Tatiana von Landesberger
+
+**Preserving Minority Structures in Graph Sampling**
+<br/>
+Authors: Ying Zhao, Haojin Jiang, Qi’an Chen, Yaqi Qin, Huixuan Xie, Yitao Wu, Shixia Liu, Zhiguang Zhou, Jiazhi Xia, and Fangfang Zhou
+
+## InfoVis
+
+### Best Paper Award
+
+**Visual Reasoning Strategies and Satisficing: How Uncertainty Visualization Design Impacts Effect Size Judgments and Decisions**
+<br/>
+Authors: Alex Kale, Matthew Kay, and Jessica Hullman
+
+### Best Paper Honorable Mention
+
+**A Design Space of Vision Science Methods for Visualization Research**
+<br/>
+Authors: Madison Elliott, Christine Nothelfer Nielsen, Cindy Xiong, and Danielle Albers Szafir
+
+
+**A Simple Pipeline for Coherent Grid Maps**
+<br/>
+Authors: Wouter Meulemans, Max Sondag, and Bettina Speckmann
+
+
+**A Generic Framework and Library for Exploration of Small Multiples through Interactive Piling**
+<br/>
+Authors: Fritz Lekschas, Xinyi Zhou, Wei Chen, Nils Gehlenborg, Benjamin Bach, and Hanspeter Pfister
+
+
+**Data Visceralization: Enabling Deeper Understanding of Data Using Virtual Reality**
+<br/>
+Authors: Benjamin, David Brown, Bongshin Lee, Christophe Hurter, Steven Drucker, and Tim Dwyer 
```

Comment:
Benjamin is missing a last name, perhaps?

Reply:
Yes, left it for InfoVis, since I'm not sure. I sent them a note.

Reply:
kk we'll have some time before we link this page up, so we'll have some time :)